client/packages/host/src/Admin/ElectronSettings.tsx

```diff
@@ -6,6 +6,7 @@ import {
   Box,
   LoadingButton,
   LocaleKey,
+  Switch,
```

Comment:
Using switch, same as request edit modal (when selection order in quantity or packs)